packages/suite-data/files/translations/cs.json

```diff
@@ -1224,6 +1224,7 @@
   "TR_NAV_TOKENS": "Tokeny",
   "TR_NAV_TRADE": "Obchod",
   "TR_NAV_TRANSACTIONS": "Přehled",
+  "TR_NEEDS_ATTENTION_NEW_DEVICE": "Připojen nový trezor.",
```

Comment:
@jvaclavik should I also fill up czech translations?

Reply:
No, you need to fill up only translations here: `packages/suite/src/support/messages.ts`

Everything else is auto-generated. So `cs.json` and `en.json` will be overwritten, please update it in `messages.ts`